spec.bs

```diff
@@ -4672,6 +4672,33 @@ An <dfn>ad size</dfn> is the Width and height of an ad.
   :: A [=string=]. Can only be one of "px" (pixel), "sh" (screen height), and "sw" (screen width).
 </dl>
 
+<h3 id=direct-from-seller-signals-section>Direct from seller signals</h3>
+
+A <dfn>direct from seller signals key</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="direct from seller signals key">
+  : <dfn>seller</dfn>
+  :: An [=origin=]. Matches the origin that served the captured [:Ad-Auction-Signals:] header.
+  : <dfn>ad slot</dfn>
+  :: A [=string=]. Matches the `adSlot` key of the JSON dictionaries in the top-level array of the
+    [:Ad-Auction-Signals:] value.
+</dl>
+
+A <dfn>direct from seller signals</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="direct from seller signals">
+  : <dfn>auction signals</dfn>
+  :: Null or a [=string=].
+    Opaque JSON data passed to both buyers' and the seller's [=script runners=].
+  : <dfn>seller signals</dfn>
+  :: Null or a [=string=].
+    Opaque JSON data passed to the seller's [=script runner=].
+  : <dfn>per buyer signals</dfn>
+  :: A [=map=] whose [=map/keys=] are [=origins=] and whose [=map/values=] are [=strings=].
+    [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are opaque JSON data

```

Comment:
Let's link to `[=origin=]`. In fact let's make this more like https://github.com/WICG/turtledove/pull/881/files?diff=split&w=1#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR4409-R4410 where we link to "scheme" and write out the literal "https".